src/borg/_hashindex.c

```diff
@@ -56,8 +56,10 @@ typedef struct {
     int lower_limit;
     int upper_limit;
     int min_empty;
+#ifdef Py_PYTHON_H
```

Comment:
I dislike depending on internal things like that when avoidable. The cleanest would be an explicit define HASHINDEX_WITHOUT_PYTHON (set explicitly in the testing / fuzzing compilation). But picking something from the public api would still be better.